samples/Azure.Storage.Tables/Generated/Models/CorsRule.cs

```diff
@@ -18,7 +18,7 @@ public partial class CorsRule
         /// <param name="allowedHeaders"> the request headers that the origin domain may specify on the CORS request. </param>
         /// <param name="exposedHeaders"> The response headers that may be sent in the response to the CORS request and exposed by the browser to the request issuer. </param>
         /// <param name="maxAgeInSeconds"> The maximum amount time that a browser should cache the preflight OPTIONS request. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="allowedOrigins"/>, <paramref name="allowedMethods"/>, <paramref name="allowedHeaders"/>, or <paramref name="exposedHeaders"/> is null. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="allowedOrigins"/>, <paramref name="allowedMethods"/>, <paramref name="allowedHeaders"/> or <paramref name="exposedHeaders"/> is null. </exception>
```

Comment:
Nit below: Why not Argument.AssertNotNull?

Reply:
It is a model. Model writer has not been updated yet.